### ISSUE TITLE
yvm 0.9.9 fails with path being undefined

### ISSUE THREAD

**Issue by torinthiel** [tophat/yvm#229]
**Describe the bug**
After upgrading to v0.9.9 some commands stop working, with error message
```
path.js:39
    throw new ERR_INVALID_ARG_TYPE('path', 'string', path);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at assertPath (path.js:39:11)
    at Object.join (path.js:1155:7)
    at setDefaultVersion (/home/torinthiel/.yvm/yvm.js:1:3632)
    at Command.<anonymous> (/home/torinthiel/.yvm/yvm.js:1:11345)
    at Command.listener (/home/torinthiel/.yvm/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:182:13)
    at Command.parseArgs (/home/torinthiel/.yvm/node_modules/commander/index.js:651:12)
    at Command.parse (/home/torinthiel/.yvm/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/home/torinthiel/.yvm/yvm.js:1:11802)
    at r (/home/torinthiel/.yvm/yvm.js:1:172)
```

**To Reproduce**
1. Install version 0.9.9 or 0.9.10
2. Run command `yarn set-default 1.12.3`
3. See error

**Expected behavior**
No error message

**Desktop (please complete the following information):**
 - OS: linux
 - YVM Version: 0.9.9
 - Shell: bash
 - Node version: v10.14.1

**Additional context**
Many commands work correctly, e.g. ls, ls-remote, install. `yvm use` works correctly if either a version number is given, or `.yvmrc` is present somewhere in directory tree, but does not work when directly in $HOME. `yvm set-default` and `yvm get-default` fail unconditionally, as does sourcing `~/.yvm/yvm.sh` which happens e.g. from `~/.bashrc`.

**Comment by torinthiel** [comment]
Happens because 50a7aca added a circular dependency between `src/util/util.js` and `src/utils/version.js`. The latter was already importing `yvmPath` and the commit added importing `getDefaultVersion` which depends on `yvmPath`). Not sure why it passes test, as both `setDefaultVersion` and `getDefaultVersion` currently fail in real script, unconditionally on first line of respective method, and the tests run both, passing correctly.

**Comment by jakebolam** [comment]
Fixed temp by reverting commit in #235 
In release v0.9.13 https://github.com/tophat/yvm/releases/tag/v0.9.13
